File: pbweberogwcl/src/app/registro-controlli/commons/dto/esito-salva-irregolarita-dto.ts
/*
 * Copyright Regione Piemonte - 2024 
 * SPDX-License-Identifier: EUPL-1.2 
*/

/* tslint:disable */
import { MessaggioDTO } from '../dto/messaggio-dto';
export interface EsitoSalvaIrregolaritaDTO {
  esito?: boolean;
  idIrregolarita?: number;
  msgs?: Array<MessaggioDTO>;
}

File: pbservizit/pbservizit-jar/src/main/java/it/csi/pbandi/pbservizit/pbandisrv/dto/irregolarita/EsitoSalvaIrregolaritaDTO.java
/*******************************************************************************
* Copyright Regione Piemonte - 2024
* SPDX-License-Identifier: EUPL-1.2
******************************************************************************/

package it.csi.pbandi.pbservizit.pbandisrv.dto.irregolarita;

public class EsitoSalvaIrregolaritaDTO implements java.io.Serializable {
	static final long serialVersionUID = 1;
	private java.lang.Boolean esito;
	private MessaggioDTO[] msgs;
	private Long idIrregolarita;
	public java.lang.Boolean getEsito() {
		return esito;
	}
	public void setEsito(java.lang.Boolean esito) {
		this.esito = esito;
	}
	public MessaggioDTO[] getMsgs() {
		return msgs;
	}
	public void setMsgs(MessaggioDTO[] msgs) {
		this.msgs = msgs;
	}
	public Long getIdIrregolarita() {
		return idIrregolarita;
	}
	public void setIdIrregolarita(Long idIrregolarita) {
		this.idIrregolarita = idIrregolarita;
	}

	
}
